roles/deploy/hooks/finalize-after.yml

```diff
@@ -11,22 +11,25 @@
       chdir: "{{ deploy_helper.current_path }}"
     when: project.update_db_on_deploy | default(update_db_on_deploy)
 
-  - name: Get WP theme template root
-    command: wp option get template_root --skip-plugins --skip-themes
+  - name: Get WP theme template and stylesheet roots
+    command: wp option get {{ item }} --skip-plugins --skip-themes
     args:
       chdir: "{{ deploy_helper.current_path }}"
     register: wp_template_root
     changed_when: false
     failed_when: wp_template_root.stderr != ""
+    with_items:
+      - template_root
+      - stylesheet_root
 
   - name: Update WP theme paths
-    command: wp option set {{ item }} {{ deploy_helper.new_release_path }}/web/wp/wp-content/themes
+    command: >
+      wp option set {{ item.item }}
+      {{ item.stdout | regex_replace(deploy_helper.releases_path + '/[^/]+(.*)', deploy_helper.new_release_path + '\1') }}
```

Comment:
I figured users could have `template_root` values with release paths different from `previous_release_path` but that still needed to be replaced. One example might be if a user imports a DB from staging where the `previous_release_path` was different than here in production. Or, perhaps the prior release was failed and `template_root` never updated but deploy helper has that failed path as `previous_release_path`? I didn't test if this latter could happen.